src/DurationDatePicker.elm

```diff
@@ -2,6 +2,7 @@ module DurationDatePicker exposing
     ( DatePicker, init, view, subscriptions
     , Settings, defaultSettings
     , openPicker, closePicker
+    , Status(..)
```

Comment:
Instead of exposing the `Status` type we should add a function that returns if the picker is open or not.
```elm
isOpen : DatePicker -> Bool
``` 
This makes it easier to change the internal state later.